### PR TITLE
Do not add an empty menu item

### PR DESCRIPTION
**Pull request by gela-d**
Before

After

### CODE REVIEW

File: src/blocks/common.blocks/menu-list/_type/menu-list_type_default.styl [hunk 1]
```diff
@@ -68,6 +53,27 @@
...
+            position: absolute;
+            top: 0;
+            right: 0;
+            left: 0;
+
+            height: 1px;
+
+            content: '';
+
+            background: -webkit-linear-gradient(left,  rgba(0,0,0,.15) 20%,#fff 100%);
```
Comment (tadatuta) [inline src/blocks/common.blocks/menu-list/_type/menu-list_type_default.styl]: autoprefixer will do everything for you )

File: src/blocks/common.blocks/post/__categories/post__categories.bemhtml [hunk 2]
```diff
@@ -10,7 +10,11 @@ block('post').elem('categories')(
             buf = [];
 
         breadcrumbs.forEach(function(item, index) {
-            var crumb = { block: 'link', content: item.title[this.ctx.lang] };
+            if(item.title) {
```
Comment (tadatuta) [inline src/blocks/common.blocks/post/__categories/post__categories.bemhtml]: `if(!item.title) return;`

**Comment by gela-d** [comment]
@tadatuta Thank u, done

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**